spec/api/people_endpoints_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'api_spec_helper'
+require 'api/people_endpoints'
+
+using Corefines::Enumerable::map_to
+
+describe API::PeopleEndpoints do
+  include_context 'API response'
+
+  subject { body }
+
+  describe 'GET /people' do
+
```

Comment:
Yeah, I guess `pending` or perhaps I should remove it: https://github.com/cvut/sirius/pull/138/files#diff-abaa46834f1db586b22dda0666c459a8R18